oneibl/registration.py

```diff
@@ -59,7 +59,7 @@ def register_dataset(file_list, one=None, created_by=None, repository=None, serv
     :return:
     """
     if created_by is None:
-        created_by = one.alyx.user
+        created_by = one._par.ALYX_LOGIN
```

Comment:
@oliche this isn't going to work with the new ONE.  Did you change it back because you ran it with a oneibl.One instance?  If so for your legacy code call register_file with the created_by argument.